gradle-jdks/src/main/java/com/palantir/gradle/jdks/AzulZuluJdkDistribution.java

```diff
@@ -16,11 +16,18 @@
 
 package com.palantir.gradle.jdks;
 
+import com.google.common.collect.ImmutableSet;
 import com.palantir.gradle.jdks.JdkPath.Extension;
 import com.palantir.gradle.jdks.JdkRelease.Arch;
+import java.util.Arrays;
+import java.util.List;
 import org.immutables.value.Value;
 
 final class AzulZuluJdkDistribution implements JdkDistribution {
+    // See https://docs.azul.com/core/zulu-openjdk/versioning-and-naming for flag details.
+    private static final ImmutableSet<String> BUNDLE_FLAGS =
+            ImmutableSet.of("ea", "embvm", "cp1", "cp2", "cp3", "c2", "criu", "cr", "crac");
```

Comment:
There's a `criu` version of the JDK?? Interesting....

Reply:
worth saying: some of these flags are discontinued! (embvm is one). One only works for ARM. The docs are very clear that just because a flag exists doesn't necessarily mean you can acquire a JVM with it set!